superset/viz.py

```diff
@@ -460,8 +462,19 @@ def get_df_payload(self, query_obj=None, **kwargs):
                     is_loaded = True
             except Exception as ex:
                 logger.exception(ex)
-                if not self.error_message:
-                    self.error_message = "{}".format(ex)
+
+                error = dataclasses.asdict(
+                    SupersetError(
+                        message="{}".format(ex),
```

Comment:
Why not `message=str(ex)`?

Reply:
this was copy pasted from before, i assume `str(ex)` works the same way, so i'll change it